### PR TITLE
(docs) Deployment: change Zeit Now → Vercel

### PR DESCRIPTION
**Pull request by hugginsio**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #38141 

## What is the new behavior?
Show correct product name in docs.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by sonukapoor — APPROVED** [review]

File: aio/content/guide/deployment.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ In the table below, you can find a list of packages which implement deployment f
 |---------------------------------------------------------------|--------------------------------------------------------------------------------|
 | [Firebase hosting](https://firebase.google.com/docs/hosting)  | [`@angular/fire`](https://npmjs.org/package/@angular/fire)                     |
 | [Azure](https://azure.microsoft.com/en-us/)                   | [`@azure/ng-deploy`](https://npmjs.org/package/@azure/ng-deploy)               |
-| [Now](https://zeit.co/now)                                    | [`@zeit/ng-deploy`](https://npmjs.org/package/@zeit/ng-deploy)                 |
+| [Vercel](https://vercel.com/home)                             | [`@vercel/ng-deploy-vercel`](https://npmjs.org/package/@zeit/ng-deploy)        |
```
Comment (IgorMinar) [inline aio/content/guide/deployment.md:79]: `@vercel/ng-deploy-vercel` is not a thing yet. do you know what are the plans to rename the packages? I'm guessing that it's just a matter of time.

**Review by IgorMinar — CHANGES_REQUESTED** [review]
Thanks for the update. This update seems exciting! https://vercel.com/blog/zeit-is-now-vercel
 
Let's wait for the npm package to be renamed, or update the PR to still use the old npm package name for now.

Otherwise this PR lgtm. thanks again.

File: aio/content/guide/deployment.md (hunk 1 above)
Comment (kjhx) [reply aio/content/guide/deployment.md:79]: I'll fix the mistake on renaming the package without changing the link. My bad.

Vercel's most common packages have already been migrated, and they'll stop publishing to the `@zeit` namespace on January 1st, 2021. I imagine the move will happen then. There's an open issue asking that very question and it's awaiting a response from a team member.

**Comment by kapunahelewong** [comment]
Should we wait until January 2021 per this note? I'm not familiar with the npm renaming process. https://www.npmjs.com/package/now

**Pull request closed without merging**